lib/engine/game/g_1849/game.rb

```diff
@@ -83,43 +92,55 @@ class Game < Game::Base
             train_limit: 2,
             tiles: %i[yellow green brown],
             operating_rounds: 3,
-            status: ['gray_uses_black'],
+            status: %w[gray_uses_black e_tokens_available],
```

Comment:
`e_tokens_available` needs to be conditional on that optional rule being enabled.